sdk/src/net.rs

```diff
@@ -1 +1,3 @@
-pub const DEFAULT_TPU_COALESCE_MS: u64 = 5;
+use std::time::Duration;
+
+pub const DEFAULT_TPU_COALESCE: Duration = Duration::from_millis(5);
```

Comment:
For backward compatibility for folks using the SDK -- can we retain DEFAULT_TPU_COALESCE_MS and use that to initialize DEFAULT_TPU_COALESCE? Similar below.

Reply:
These constants are implementation details and very specific to solana code/implementation. They are not part of the protocol specification or the api.
I don't see any reason someone should be relying on these constants and searching on github, I cannot find any such cases.
If you know of any specific instances that use these constants, then I am ok with retaining the old ones with a `deprecated` annotation; Otherwise I would rather not to bloat the code unnecessarily.